NAMESPACE

```diff
@@ -141,6 +146,7 @@ export(use_template)
 export(use_test)
 export(use_testthat)
 export(use_tibble)
+export(use_tidy_actions)
```

Comment:
Ditto re: not having same prefix as the rest.

Reply:
This is meant to be consistent with `use_tidy_ci()`.

Reply:
Do you think the GitHub action stuff should *be* `use_tidy_ci()` now?